languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -41914,6 +41914,32 @@ though, and has since been slightly modified)-->
         </rule>
     </category>
     <category id="CAT_TOURS_CRITIQUES" name="Tours critiqués divers (barbarismes, impropriétés, solécismes, etc.)">
+        <rulegroup id="DE_MA_FAUTE" name= "utiliser -ma faute- plutôt que -de ma faute-">
```

Comment:
The standard formatting for the name would be, in this case, `name="Ce n'est pas de ma faute (ma faute)"` (careful not to put any unwarranted spaces in the syntax)